server/controllers/v1/tests/user.test.ts

```diff
@@ -143,6 +145,71 @@ describe('v1 - User Controller', () => {
       test('Password should not be included in serialized result', () => {
         expect(signInResponse.data.result.password).not.toBeDefined();
       });
+
```

Comment:
Add test: Other users should not be affected if someone is locked out.

(User 1 is locked out, user 2 can still login normally)